dockerfiles/Dockerfile.cross

```diff
@@ -1,5 +1,5 @@
 
-FROM    golang:1.8.1
+FROM    golang:1.8.3
```

Comment:
Then I'd suggest making sure all `Dockerfile`s are locked into a specific golang version. The 3 files in `dockerfiles` dir.